jax/_src/random.py

```diff
@@ -2030,6 +2030,8 @@ def orthogonal(
 ) -> Array:
   """Sample uniformly from the orthogonal group O(n).
 
+  Uses the technique described in https://arxiv.org/abs/math-ph/0609050.
```

Comment:
Did you mean to include this reference as part of this PR?

Reply:
Yeah, just fixing a small omission to my [earlier PR](https://github.com/google/jax/pull/10458).

I can submit it as a separate PR, if desired.

Reply:
Thanks for the clarification! Yeah, I think it would be better to open that as a separate PR if it's not too much trouble. Please ping me on that PR and I'll handle it!